src/main/kotlin/com/evalify/evalifybackend/quiz/domain/DTO/QuizStudentDTO.kt

```
@@ -0,0 +1,5 @@
+package com.evalify.evalifybackend.quiz.domain.DTO
+
+class QuizStudentDTO (
+    val ipAddress:String
+    ){}
```

detekt (1.23.8)

[warning] The class or object QuizStudentDTO is empty.

(detekt.empty-blocks.EmptyClassBlock)